packages/plugins/typescript-resolvers/tests/mapping.spec.ts

```diff
@@ -132,16 +132,16 @@ describe('ResolversTypes', () => {
     export type ResolversTypes = {
       Query: Partial<Query>,
       MyType: Partial<MyType>,
-      String: Scalars['String'],
+      String: Partial<Scalars['String']>,
```

Comment:
One small change: if `defaultMapper` contains the `{T}` we apply it even on scalars. I think it makes sense because `Scalars['ID']` might be anything, a primitive but also an object.

Reply:
It's not always `Partial<{T}>`, maybe someone wants to wrap it like this `{T} | Wrapped<{T}>`.